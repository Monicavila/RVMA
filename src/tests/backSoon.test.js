import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { act } from "react-dom/test-utils";
import BackSoon from "../components/landing/landing";

test('renders switch on your meeting', () => {
    act(()=>{
    render(<BackSoon />);
    })
    const linkElement = screen.getByTestId("switch_off");
    expect(linkElement).toBeInTheDocument();
});

it("renders the Landing snapshot", () => {
    const tree = renderer.create(<BackSoon />).toJSON();
    expect(tree).toMatchSnapshot();
});