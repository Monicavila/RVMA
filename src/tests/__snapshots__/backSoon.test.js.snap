// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Landing snapshot 1`] = `
<div
  className="Landing_page"
>
  <div
    className="left_side"
  />
  <div
    className="center_down"
  >
    <div
      className="slider"
    >
      <div
        className="switch_button"
      >
        <input
          className="switch_button__checkbox"
          data-testid="switch_off"
          id="switch_label"
          name="switch_button"
          onClick={[Function]}
          type="checkbox"
        />
        <label
          className="switch_button__label"
          htmlFor="switch_label"
        />
      </div>
    </div>
  </div>
  <div
    className="right_side"
  >
    <div
      className="logo"
    />
    <div
      className="startmeeting"
    />
  </div>
</div>
`;
